builder/builder.proto

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// Package builder provides protocol buffer versions of some of the top-level
```

Comment:
GA4GH has largely standardized on JSON for interchange.  Should we just drop the protos and add JSON tags to the in memory types?

Reply:
I agree on JSON for interchange, but this is for local configuration and never on the API boundary.  To rebuild the oneof usage in JSON would require writing string matching and selecting which struct to JSON-unmarshal on top of, so I think it's much cleaner to keep this as proto.